markdown/dev/howtos/ways-to-contribute/technical-writing/patterns/en.md

```diff
@@ -2,6 +2,12 @@
 title: Writing for freesewing.org
 ---
 
-You could write documentation for freesewing.org, our makers website.
+A good way of contributing to FreeSewing  is writting documentation! 
 
-You would need good writing skills and a familiarity with sewing.
+We don't have specific guidelines on how to write documentation. You can write as you feel fit as long as it's relevant and follows [our Code of Conduct](/guides/code-of-conduct/) and [our community standards](https://freesewing.org/docs/various/community-standards/).
+
+Once in a while you might see some documents using personal tone, such as `When pressing a seam, I like to lay it flat and ensure there are no folds ...`. While the impersonal tone is more common for documentation, the personal tone shows that FreeSewing is not a souless entity, but instead we are a community of awesome people! 
+
+Whenever changing existing documents (be it extending, correcting errors, etc) try to preserve the tone. Don't rewrite the documents to solely change the tone.
```

Comment:
SeaZeeZee mentioned getting in touch with the authors too and I don't want that to feel like the authors are gatekeepers of documents they've written. There is another facet to it too: authors may not be involved in the project anymore so they wouldn't be reachable. 

However I concede that trying to reach out is a good idea but wouldn't want to have that come through as `the author has to approve any changes you want to make to documentation they've written.`

I'll give it a thought on how to phrase that and will update the content.

Reply:
The outcome I understood from the call last Saturday was that we would like to try and 'preserve' the tone of existing documentation rather than rewriting it in a different tone. (Or like Joost suggested, you could add your own opinion/thoughts in a comment.) I understand not wanting to give the impression that documentation on the site is final because its definitely not! But I do think we need something to indicate that that is what we would like to happen with current documentation. If that makes any sense at all.

Reply:
I agree with that! However that was not what I intended to express on my previous comment. My Bad.

> Saturday was that we would like to try and 'preserve' the tone of existing documentation rather than rewriting it in a different tone.

I agree. A portion of what I've written mentions that.

> I understand not wanting to give the impression that documentation on the site is final because its definitely not! But I do think we need something to indicate that that is what we would like to happen with current documentation. If that makes any sense at all.

I am not sure I understood this portion. My understanding of the portion above is that we don't want documentation to be final on the site, but we want the current one to be final (for the time being, maybe? If so, I'm confused). That's the part I am not sure I got right. 
Could you help me understand it?

Reply:
I've applied the fix suggested here + adding some sentences to nudge the person to get in touch with the author in cases where extensive changes to documentation may be required.

Let me know if this is fine. If so, @bobgeorgethe3rd could you resolve the conversation?